physics/cu_gf_deep.F90

```diff
@@ -28,8 +28,8 @@ module cu_gf_deep
      real(kind=kind_phys), parameter :: pgcd = 0.1
 !
 !> aerosol awareness, do not user yet!
```

Comment:
```
do not use yet!
```

Reply:
I found this in few lines (1561, 1564, 1607, 1611)